src/aiida/brokers/rabbitmq/broker.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, profile: Profile) -> None:
         self._prefix = f'aiida-{self._profile.uuid}'
 
     def __str__(self):
-        return f'RabbitMQ v{self.get_rabbitmq_version()} @ {self.get_url()}'
+        try:
+            return f'RabbitMQ v{self.get_rabbitmq_version()} @ {self.get_url()}'
+        except ConnectionError:
+            return f'RabbitMQ @ {self.get_url()}'
```

Comment:
Maybe we should include here that the connection could not be established?
```suggestion
            return f'RabbitMQ @ {self.get_url()}: Connection failed'
```
or something like that

Reply:
Hmm, sure, not a bad idea. A bit of bike shedding: do we use `: Connection failed` or `<Connection failed>`? I kind of favor the latter in a string representation. In the `verdi status` command it would look like:

```python
 ✘ broker:      Unable to connect to broker: RabbitMQ @ amqp://guest:guest@127.0.0.1:5672?heartbeat=600 <Connection failed>
Error: ConnectionError: [Errno 61] Connect call failed ('127.0.0.1', 5672)
```

vs

```python
 ✘ broker:      Unable to connect to broker: RabbitMQ @ amqp://guest:guest@127.0.0.1:5672?heartbeat=600: Connection Failed
Error: ConnectionError: [Errno 61] Connect call failed ('127.0.0.1', 5672)
```

Reply:
sure, your suggestion looks fine to me